providers/implementations/kdfs/tls1_prf.c

```diff
@@ -181,9 +181,6 @@ static int kdf_tls1_prf_set_ctx_params(void *vctx, const OSSL_PARAM params[])
     }
     /* The seed fields concatenate, so process them all */
     if ((p = OSSL_PARAM_locate_const(params, OSSL_KDF_PARAM_SEED)) != NULL) {
-        OPENSSL_cleanse(ctx->seed, ctx->seedlen);
```

Comment:
Has this line moved to somewhere else??

Reply:
Not exactly. It gets reset anyway as part of kdf_tls1_prf_reset(). Prior to this the semantics of the `OSSL_KDF_PARAM_SEED` parameter was that you could supply multiple values in a single set_ctx_params call and they would get concatenated together. However, if you supplied it again in a subsequent `set_ctx_params` call then anything you had supplied previously would be wiped and we start again. After this change any seed values you supply will always be concatenated together - regardless of whether you supply them in a single `set_ctx_params` call or not. This is required for backwards compatibility because previously (via the bridge) these were supplied via ctrl calls - which doesn't have the concept of batching multiple values up in one go.